webapp/scripts/functions.js

```diff
@@ -269,16 +314,31 @@ function parseDescription(raw) {
  * @param {*} mapset mapset name
  * @param {*} table table name
  */
-function addBounds(desc, mapset, table) {
-  desc.columnObj.rows.forEach(row => {
-    if (['DOUBLE PRECISION', 'INTEGER'].indexOf(row.type) > -1) {
-      const [min, max] = module.exports.getUnivarBounds(mapset, table, row.column)
-      row.min = min
-      row.max = max
+function setBounds(desc, mapset, table) {
+  let i = 0
+  while (i < desc.columnObj.rows.length) {
+    const bounds = module.exports.getUnivarBounds(mapset, table, desc.columnObj.rows[i].column)
+    // add bounds when they exist, remove element when they don't
+    if (bounds.length) {
+      desc.columnObj.rows[i].min = bounds[0]
+      desc.columnObj.rows[i].max = bounds[1]
+      i++
+    } else {
+      desc.columnObj.rows.splice(i, 1)
     }
-  })
+  }
+}
+function addDescription(desc, table) {
+  if (metadata.length) {
+    const meta = metadata.filter(m => m.table === table)[0]
+    if (meta != undefined) {
```

Comment:
Same as `if (meta) { }`. If you want to test for `undefined` instead of falsy, use `!==`